contracts/MirrorNodeFFI.sol

```diff
@@ -131,6 +130,20 @@ contract MirrorNodeFFI is MirrorNode {
         }
     }
 
+    function _surl(string memory url) private returns (uint256 status, bytes memory data) {
```

Comment:
let's move this to a library in the `contracts/` folder, also called Surl as before, and this method could be called `get` (`internal` method). In the library include a NatSpec comment referencing Surl and its url https://github.com/memester-xyz/surl, and the reason we decided to inline the library instead of using it directly.